recipes/mongomock/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{%set name = "mongomock" %}
+{%set version = "3.6.0" %}
+{%set build_num = "0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "2e6b85f97b84d1797c5b0deceff72690a2e79f82c8ec410663a4175a83b8a80f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.{{ compress_type }}
+  url: https://github.com/mongomock/mongomock/archive/{{ version }}.{{ compress_type }}
```

Comment:
[PyPI](https://pypi.org/project/mongomock/#files) only has a wheel, unfortunately. This is definitely a bit odd -  there's certainly an sdist at [the url].(https://github.com/mongomock/mongomock/archive/3.6.0.tar.gz). I'm kind of flummoxed. Maybe an issue with `pbr`? ([reference](https://github.com/ekarlso/python-odlclient/issues/3))